pyproject.toml

```diff
@@ -1,6 +1,260 @@
+"core" = "aiida.orm.groups:Group"
+"core.auto" = "aiida.orm.groups:AutoGroup"
+"core.import" = "aiida.orm.groups:ImportGroup"
+"core.upf" = "aiida.orm.groups:UpfFamily"
+
+[project.entry-points."aiida.node"]
+"data" = "aiida.orm.nodes.data.data:Data"
+"process" = "aiida.orm.nodes.process.process:ProcessNode"
+"process.calculation" = "aiida.orm.nodes.process.calculation.calculation:CalculationNode"
+"process.calculation.calcfunction" = "aiida.orm.nodes.process.calculation.calcfunction:CalcFunctionNode"
+"process.calculation.calcjob" = "aiida.orm.nodes.process.calculation.calcjob:CalcJobNode"
+"process.workflow" = "aiida.orm.nodes.process.workflow.workflow:WorkflowNode"
+"process.workflow.workchain" = "aiida.orm.nodes.process.workflow.workchain:WorkChainNode"
+"process.workflow.workfunction" = "aiida.orm.nodes.process.workflow.workfunction:WorkFunctionNode"
+
+[project.entry-points."aiida.parsers"]
+"core.arithmetic.add" = "aiida.parsers.plugins.arithmetic.add:ArithmeticAddParser"
+"core.templatereplacer.doubler" = "aiida.parsers.plugins.templatereplacer.doubler:TemplatereplacerDoublerParser"
+
+[project.entry-points."aiida.schedulers"]
+"core.direct" = "aiida.schedulers.plugins.direct:DirectScheduler"
+"core.lsf" = "aiida.schedulers.plugins.lsf:LsfScheduler"
+"core.pbspro" = "aiida.schedulers.plugins.pbspro:PbsproScheduler"
+"core.sge" = "aiida.schedulers.plugins.sge:SgeScheduler"
+"core.slurm" = "aiida.schedulers.plugins.slurm:SlurmScheduler"
+"core.torque" = "aiida.schedulers.plugins.torque:TorqueScheduler"
+
+[project.entry-points."aiida.transports"]
+"core.local" = "aiida.transports.plugins.local:LocalTransport"
+"core.ssh" = "aiida.transports.plugins.ssh:SshTransport"
+
+[project.entry-points."aiida.tools.calculations"]
+
+[project.entry-points."aiida.tools.dbexporters"]
+
+[project.entry-points."aiida.tools.dbimporters"]
+"core.cod" = "aiida.tools.dbimporters.plugins.cod:CodDbImporter"
+"core.icsd" = "aiida.tools.dbimporters.plugins.icsd:IcsdDbImporter"
+"core.materialsproject" = "aiida.tools.dbimporters.plugins.materialsproject:MaterialsProjectImporter"
+"core.mpds" = "aiida.tools.dbimporters.plugins.mpds:MpdsDbImporter"
+"core.mpod" = "aiida.tools.dbimporters.plugins.mpod:MpodDbImporter"
+"core.nninc" = "aiida.tools.dbimporters.plugins.nninc:NnincDbImporter"
+"core.oqmd" = "aiida.tools.dbimporters.plugins.oqmd:OqmdDbImporter"
+"core.pcod" = "aiida.tools.dbimporters.plugins.pcod:PcodDbImporter"
+"core.tcod" = "aiida.tools.dbimporters.plugins.tcod:TcodDbImporter"
+
+[project.entry-points."aiida.tools.data.orbitals"]
+"core.orbital" = "aiida.tools.data.orbital.orbital:Orbital"
+"core.realhydrogen" = "aiida.tools.data.orbital.realhydrogen:RealhydrogenOrbital"
+
+[project.entry-points."aiida.workflows"]
+"core.arithmetic.multiply_add" = "aiida.workflows.arithmetic.multiply_add:MultiplyAddWorkChain"
+"core.arithmetic.add_multiply" = "aiida.workflows.arithmetic.add_multiply:add_multiply"
+
+[tool.flit.module]
```

Comment:
How and where is the functionality of the `MANIFEST.in` taken care off?

Reply:
flit already handles all this out-the-box, see e.g. https://flit.readthedocs.io/en/latest/pyproject_toml.html#sdist-section.
If you want to check what is included in the sdist and wheel, feel free to try running `flit build` and exploring the distributions yourself. I've already done this, though, and can confirm that all `package_data`is correctly in the wheel and sdist.

Reply:
How does it know _what_ files to include though? Does it simply include all files within the `aiida` folder? The docs you link say:

> When you use flit build or flit publish, Flit builds an sdist (source distribution) tarball containing the files that are checked into version control (git or mercurial). 

That seems to suggest to me that it would also include the `tests` folder for example. Is that true? Because I don't think that is desirable. This would make the package unnecessarily heavy, and we intentionally moved the tests outside of the package such that they don't have to be shipped with the build.